homeassistant/components/fibaro/__init__.py

```diff
@@ -453,59 +453,59 @@ def __init__(self, fibaro_device: DeviceModel) -> None:
         if not fibaro_device.visible:
             self._attr_entity_registry_visible_default = False
 
-    async def async_added_to_hass(self):
+    async def async_added_to_hass(self) -> None:
         """Call when entity is added to hass."""
         self.controller.register(self.fibaro_device.fibaro_id, self._update_callback)
 
-    def _update_callback(self):
+    def _update_callback(self) -> None:
         """Update the state."""
         self.schedule_update_ha_state(True)
 
     @property
-    def level(self):
+    def level(self) -> int | None:
         """Get the level of Fibaro device."""
         if self.fibaro_device.value.has_value:
             return self.fibaro_device.value.int_value()
         return None
 
     @property
-    def level2(self):
+    def level2(self) -> int | None:
         """Get the tilt level of Fibaro device."""
         if self.fibaro_device.value_2.has_value:
             return self.fibaro_device.value_2.int_value()
         return None
 
-    def dont_know_message(self, action):
+    def dont_know_message(self, action: str) -> None:
```

Comment:
`action` seems to be unused. Is that intentional?

Reply:
You're right. I updated the PR.